File: adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/interpreter/InterpreterParameterDescription.java
package org.jvalue.ods.adapterservice.adapter.interpreter;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class InterpreterParameterDescription {
  private String name;
  private String description;
  private Class<?> type;
}

File: adapter/src/adapter/interpreter/InterpreterParameterDescription.ts
export class InterpreterParameterDescription{

    name: string;
    description: string;
    required: boolean | undefined;
    type: unknown;

    constructor (name: string, description: string, type: unknown) {
        this.name = name;
        this.description = description;
        this.type  = type;
    }
}